src/Ch9/Ch9.Shared/App.xaml.cs

```diff
@@ -179,6 +179,45 @@ private void ConfigureOrientation()
 			{
 				DisplayInformation.AutoRotationPreferences = DisplayOrientations.Portrait;
 			}
+
+			void OnOrientationChanged(DisplayInformation displayInformation, object args)
+			{
+				ShowViewModel showViewModel = null;
+
+				if ((_rootFrame.Content as FrameworkElement)?.DataContext is ShowPageViewModel showPage)
+				{
+					showViewModel = showPage.Show;
+				}
+				else if ((_rootFrame.Content as FrameworkElement)?.DataContext is MainPageViewModel mainPage)
+				{
+					showViewModel = mainPage.Show;
+				}
+
+				if (showViewModel == null) 
+					return;
+
+				ToVideoFullWindow(showViewModel, displayInformation);
+			}
+
+			DisplayInformation.GetForCurrentView().OrientationChanged += OnOrientationChanged;
+		}
+
+		/// <summary>
+		/// Sets the video to fullWindow depending on screen orientation
+		/// </summary>
+		/// <param name="showVm">The show vm for which we will change its property IsVideoFullWindow</param>
+		/// <param name="displayInformation">The displayInformation for the current view</param>
+		private void ToVideoFullWindow(ShowViewModel showVm, DisplayInformation displayInformation)
+		{
+			if (displayInformation.CurrentOrientation == DisplayOrientations.Landscape || 
+			    displayInformation.CurrentOrientation == DisplayOrientations.LandscapeFlipped)
+			{
+				showVm.IsVideoFullWindow = true;
+			}
+			else
+			{
+				showVm.IsVideoFullWindow = false;
```

Comment:
Isn't that going to remove the full screen if I clicked on the Fullscreen button then rotate the screen?

Reply:
Nope. Cause the full screen button is available only in portrait mode. So in that scenario, it won't cause this behavior as rotating would just set it to true again. Then, if rotating from landscape to portrait in full screen, yes it would remove the full screen, which is the expected behavior.